docs/index.rst

```diff
@@ -33,3 +33,14 @@
    supported_environments
    changelog
    licenses
+
+.. toctree::
+   :hidden:
+   :caption: Hands-on Tutorial
+
+   intro
+   contents
+   exercise_1
+   exercise_2
+   exercise_3
+   additional_features
```

Comment:
Let's actually yank this up higher - I think I'd like to create a new section called "Concepts" and drop `run` through `performance` into it, and then lift this section up to between `getting_started` and `run`. That way the sidebar will be ordered as:

- Overview
- Getting started
- Hands-on Tutorial
  - About these tutorials
  - Exercise 1 - ...
  - Exercise 2 - ...
  - Exercise 3 - ...
  - What to learn about next
- Concepts
   - run
   - python_allocators
   ...
```
I think that's a pretty nice organization, and should do a good job of highlighting these new docs. We should also make getting_started.html suggest that the next steps after you've generated your first flame graph would be to either try the tutorials or to learn more about `memray run`, with links to either the tutorial or skipping over it to the concepts section.